django-backend/fecfiler/dcf_importer/dcf_parser.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+
+parser = argparse.ArgumentParser(
+    description=""
+    "This script is a simple tool I've thrown together to poke at the format of DCF files"
+)
+parser.add_argument(
+    "dcf_filename",
+    help="the dcf file to be parsed",
+),
+parser.add_argument(
+    "row_type",
+    nargs="?",
+    help="filter to the given row type.  Default is all rows."
+)
+parser.add_argument(
+    "col_numbers",
+    nargs="*",
+    default=[],
+    help="filter to the given column.  Default is all columns."
+),
+parser.add_argument(
+    "-v", "--verbose", help="record and print minor errors", action="store_true"
+)
+parser.add_argument(
+    "-d",
+    "--debug",
+    help="Prints the names of sheets and fields as the script works",
+    action="store_true",
+)
+args = parser.parse_args()
+VERBOSE = args.verbose  # Controls whether or not checks for minor errors are run
+DEBUG = args.debug      # If true, script will print its state in great detail as it runs
+DCF_FILENAME = args.dcf_filename
+ROW_TYPE = args.row_type
+COL_NUMBERS = args.col_numbers
+
+
+# Merges extra columns that were made when a comma was included in a quotation
+def get_sanitized_row(line, row_number):
```

Comment:
found this cool implementation for people running into this problem
https://stackoverflow.com/a/21527375

Basically, python has a csv utility and you can tell it what the delimiter is and what kinds of quotes supersede the delimiter.

No need to change, but I thought you'd be interested

Reply:
Oh sick, I will keep that in mind for the future!  We will probably want to use that method when building out the actual importing, just to better cover any potential edge cases.